crates/forge_analyzer/src/definitions.rs

```diff
@@ -1021,6 +1024,49 @@ impl<'cx> FunctionAnalyzer<'cx> {
         }
     }
 
+    fn lower_jsx_attr(&mut self, n: JSXElement) {
+        n.opening.attrs.iter().for_each(|attr| match attr {
+            JSXAttrOrSpread::JSXAttr(jsx_attr) => {
+                if let JSXAttrName::Ident(ident_value) = &jsx_attr.name {
+                    if let Some(value) = &jsx_attr.value {
+                        if let JSXAttrValue::JSXExprContainer(jsx_expr) = value {
+                            self.lower_jsx_handler(jsx_expr, ident_value);
+                        }
+                    }
+                }
+            }
+            JSXAttrOrSpread::SpreadElement(_) => {}
+        });
+    }
+
+    fn lower_jsx_handler(&mut self, n: &JSXExprContainer, ident_value: &Ident) {
+        if let JSXExpr::Expr(expr) = &n.expr {
+            self.lower_expr(&expr);
+            if ident_value.sym.contains("on") {
```

Comment:
To make this more accurate, you could make sure the ident starts with "on" and the next letter is capitalized.